build-and-test.ps1

```diff
@@ -1,22 +1,27 @@
 #!/usr/bin/env pwsh
 [cmdletbinding()]
 param(
-    [ValidateSet('*', 'runtime-sdk', 'aspnet', 'wcf')]
-    [string]$ImageTypeFilter = '*',
+    [ValidateSet('*', 'runtime', 'sdk', 'aspnet', 'wcf')]
```

Comment:
It seems like '*' could be excluded.  The default could be an empty array which would indicate no filtering.  It would arguably make more sense because it wouldn't allow values like `runtime,*`.  Does `run-test` work with `runtime,*`?